data/ecosystems/s/starknet.toml

```diff
@@ -3838,17 +3838,11 @@ url = "https://github.com/l-henri/brain-dump"
 url = "https://github.com/l-henri/StarkNet-graffiti"
 
 [[repo]]
-url = "https://github.com/lambdaclass/cairo-rs"
-
-[[repo]]
-url = "https://github.com/lambdaclass/cairo-rs-py"
```

Comment:
please add https://github.com/lambdaclass/cairo-vm-py in favor of this deletion

Reply:
https://github.com/lambdaclass/cairo-vm-py is already in the list (below at line 3847)